#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/lisp-bfuncs--ignore-jobserver.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: 
## DP: Author: Jari Aalto <jari.aalto@cante.net>
## DP: 
## DP: 
## DP:     lisp/bfuncs: (edb-bfunc-get-GNUmakefile): ignore message jobserver 
## DP: unavailable

@DPATCH@

diff --git a/lisp/bfuncs b/lisp/bfuncs
index a225c21..fc23247 100644
--- a/lisp/bfuncs
+++ b/lisp/bfuncs
@@ -39,6 +39,9 @@
                (call-process-region (point-min) (point-max) "make" t t nil
                                     "--no-print-directory"
                                     "-f" "GNUmakefile" "-f" "-" "zup")
+	       (goto-char (point-min))
+	       ;; Make[4]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
+	       (delete-matching-lines "jobserver")
                (buffer-substring (point-min) (1- (point-max))))))
     (if (interactive-p)
         (message "%s" ans)
@@ -90,6 +93,7 @@
           (write-file "edbcore.el"))))))
 
 (defun edb-bfunc-make-all ()
+  (setq debug-on-error t)
   (edb-bfunc-make-edbcore)
   (let ((dir (car load-path))
         (els (edb-bfunc-get-GNUmakefile "installed-el-files")))
